### PR TITLE
Add try except around command creation/invocation.

### PR DESCRIPTION
**Pull request by russellhadley**
Adds a guard and error message around commands to make a more graceful
experience when a tool isn't on the path.

### CODE REVIEW

**Comment by russellhadley** [comment]
@AndyAyersMS - Hardens command creation and adds a more sensible error message.   Should be last element of #1.  (I'm going to open a separate issue for updating the frameworks due to reconciling the list with the .NETStandard set)

**Comment by russellhadley** [comment]
@pgavlin take a look?

File: src/jit-analyze/jit-analyze.cs [hunk 1]
```diff
@@ -546,7 +546,18 @@ public static bool DiffInText(string diffPath, string basePath)
...
-            Command diffCmd = Command.Create(@"git", commandArgs);
+            Command diffCmd = null;
+
+            try 
+            {
+                diffCmd = Command.Create(@"git", commandArgs);
+            }
+            catch (CommandUnknownException e)
+            {
+                Console.WriteLine("\nError: git command not found!  Add git to the environment path.\n", e);
```
Comment (pgavlin) [inline src/jit-analyze/jit-analyze.cs]: Please use `Console.Error`

File: src/jit-dasm/jit-dasm.cs [hunk 2]
```diff
@@ -515,9 +515,17 @@ public void GenerateAsm()
...
-                        commandArgs);
+                    Command generateCmd = null;
+
+                    try 
+                    {
+                        generateCmd = Command.Create(_executablePath, commandArgs);
+                    }
+                    catch (CommandUnknownException e)
+                    {
+                        Console.WriteLine("\nError: {0} command not found!\n", e);
```
Comment (pgavlin) [inline src/jit-dasm/jit-dasm.cs]: Same

File: src/jit-diff/jit-diff.cs [hunk 3]
```diff
@@ -556,6 +551,26 @@ public static int Main(string[] args)
...
+
+                // Wireup stdout/stderr so we can see outout.
+                command.ForwardStdOut();
+                command.ForwardStdErr();
+
+                return command.Execute();
+            }
+            catch (CommandUnknownException e)
+            {
+                Console.WriteLine("\nError: {0} command not found!  Add {0} to the path.", name, e);
```
Comment (pgavlin) [inline src/jit-diff/jit-diff.cs]: Same

**Comment by russellhadley** [comment]
@pgavlin Added separate commit for switching appropriate output to stderr.

**Comment by pgavlin** [comment]
LGTM

**Pull request merged**